src/SimulationControl/Spheral_banner.py

```diff
@@ -1 +1,3 @@
+SpheralBranch = "@abbref@"
+SpheralShortCommit = "@short@"
 print "|  %-76s|" % "Spheral version             : @spheralversion@"
```

Comment:
@jmikeowen I'm passing in the commit and branch name we generate from CMake here. Is there a better place for this to live?